lib/mailchimp_api/resources/interest_category.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module MailchimpAPI
+  class InterestCategory < Base
+    extend MailchimpAPI::Support::Countable
+
+    self.collection_parser = CollectionParsers::InterestCategory
+
+    self.prefix = '/3.0/lists/:list_id/'
+    self.collection_name = 'interest-categories'
```

Comment:
Do you also need `self.element_name = 'interest-category'` or does it infer it from this?